src/graphql.rs

```diff
@@ -1834,7 +1834,7 @@ mod tests {
             let ingest_sensors = Arc::new(tokio::sync::RwLock::new(
                 CURRENT_GIGANTO_INGEST_SENSORS
                     .into_iter()
-                    .map(|sensor| sensor.to_string())
+                    .map(&str::to_string)
```

Comment:
I don't think `&` is necessary. Function names are in general considered function pointers.

Reply:
Applied the suggested adjustment. Thank you.